libcontainer/console_linux.go

```diff
@@ -89,7 +89,14 @@ func (c *linuxConsole) mount(rootfs, mountLabel string, uid, gid int) error {
 	if f != nil {
 		f.Close()
 	}
-	return syscall.Mount(c.slavePath, dest, "bind", syscall.MS_BIND, "")
+	if err := syscall.Mount(c.slavePath, dest, "bind", syscall.MS_BIND, ""); err != nil {
+		return err
+	}
+
+	if err := syscall.Mount("", dest, "", syscall.MS_PRIVATE, ""); err != nil {
+		return err
```

Comment:
W.r.t why we are defining a new function to set mount propagation flag, I found following in mount man page.

"Note  that  the  Linux  kernel does not allow to change multiple propagation flags with a single mount(2) syscall, and the  flags cannot be mixed with other mount options."

So my understanding is that propagation property of mount point has to be set using a separate call to syscall. I think docker does the same thing.

W.r.t errors, I will put more information when error happens for easier debugging.